modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -871,6 +871,132 @@ definition {
+	test CanFilterEntries {
+		task ("Given a Custom Object and two entries (approved and pending)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 156806",
+				objectName = "CustomObject156806",
+				pluralLabelName = "Custom Objects 156806");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject156806");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject156806");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject156806",
+				value = "Entry Approved");
+
+			ObjectAdmin.openWorkflowProcessBuilderConfiguration();
+
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "Custom Object 156806");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject156806",
+				value = "Entry Pending");
+		}
+
+		task ("When add a Custom View with filters (Create Date, Modified Date, and Status)") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 156806");
+
+			ObjectCustomViews.goToViewsTab();
+
+			ObjectCustomViews.addObjectViewViaUI(viewName = "Custom Views");
+
+			ObjectCustomViews.goToViewsDetails(label = "Custom Views");
+
+			ObjectCustomViews.markViewAsDefault();
+
+			ObjectCustomViews.goToViewsBuilderTab();
+
+			var listMetaData = "Custom Field,Create Date,Modified Date,Status";
+
+			ObjectCustomViews.addColumnsViaUI(addColumns = "${listMetaData}");
+
+			ObjectCustomViews.goToFiltersTab();
+
+			ObjectCustomViews.addNewFilterViaUI(filterBy = "Create Date");
+
+			ObjectCustomViews.addNewFilterViaUI(filterBy = "Modified Date");
+
+			ObjectCustomViews.addNewFilterViaUI(
+				filterBy = "Status",
+				filterType = "Includes",
+				filterValue = "Approved,Denied,Draft,Expired,Inactive,Incomplete,In Recycle Bin,Scheduled");
+
+			Button.clickSave();
+		}
+
+		task ("Then assert filter Status is working") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject156806");
+
+			AssertElementPresent(
+				key_filter = "Approved, Denied, Draft, Expired, Inactive, Incomplete, In Recycle Bin, Scheduled",
+				locator1 = "ObjectCustomViews#LABEL_SECTION_FILTER_ENTRY");
+
+			ObjectPortlet.viewEntry(entry = "Entry Approved");
+
+			ObjectPortlet.assertEntryNotPresent(entry = "Entry Pending");
+
+			Click(
+				key_filter = "Approved, Denied, Draft, Expired, Inactive, Incomplete, In Recycle Bin, Scheduled",
+				locator1 = "ObjectCustomViews#LABEL_SECTION_FILTER_ENTRY");
+
+			Check.checkNotVisible(locator1 = "ObjectCustomViews#TOGGLE_SWITCH_EXCLUDE_STATUS");
+
+			Button.click(button = "Edit Filter");
+
+			ObjectPortlet.viewEntry(entry = "Entry Pending");
+
+			ObjectPortlet.assertEntryNotPresent(entry = "Entry Approved");
```

Comment:
This portion is for the "Exclude" toggle on the status filter, but I'm wondering if this could be another test or not? It seems a little bloated to have it all in this then block so I'm not too sure what to do about this one. @timpak can I get your suggestion?

Reply:
I agree. I'll see if there is a better way for this. Any questions I'll ask you on the slack